anlessini-qa/.gitignore

```diff
@@ -0,0 +1 @@
+mobilebert-uncased-finetuned-squadv2/
```

Comment:
What does this prototype do with SQuAD?

Reply:
Its just a normal model trained on squad, it takes in a question and context, returns a score and a beginning/end location for where the answer in the context is.